src/bundle/Resources/views/content/tabs/locations.html.twig

```diff
@@ -1,12 +1,31 @@
 {% trans_default_domain "locationview" %}
 {{ form_start(actionsForm, {'action': path('ez_hybrid_platform_ui_location_actions', {'contentId': content.id, 'redirectLocationId': location.id})}) }}
 
+<dialog id="confirm-location-removal-{{ location.id }}" class="ez-modal">
+    <section>
+        <p>{{ 'confirm.remove.location'|trans()|desc('Are you sure you want to remove selected locations?') }}</p>
+    </section>
+    <footer>
+        <button type="button" class="ez-button ez-button-secondary ez-js-close-modal">
```

Comment:
For this kind of modal (alerting the possibility of removing content) we suggested that `Cancel` buttons are styled just as `.ez-button` and not `.ez-button-secondary`